qutebrowser/test/mainwindow/conftest.py

```diff
@@ -0,0 +1,34 @@
+"""
+pytest fixtures and utilities for testing.
+
+Fixtures defined here will be visible to all test files in this directory and
```

Comment:
Sure, well noted. At the time I wrote this PR we didn't have a main `conftest.py` yet.

Let's first merge the PR (which I'm working on right now) that moves the tests outside the `qutebrowser` package first, then we can move this fixture to the main `conftest.py` file.